eclair-core/src/main/scala/fr/acinq/eclair/router/StaleChannels.scala

```diff
@@ -98,6 +87,4 @@ object StaleChannels {
     channelIsOldEnough && channelUpdateIsStale
```

Comment:
This is incorrect, this would return `false` when we don't have any channel update, which should count as a stale update. Fortunately, we have a unit test that catches this!